docs/settings/storagenetwork/_category_.json

```diff
@@ -0,0 +1,6 @@
+{
+    "position": 2,
+    "label": "Storage Network",
+    "collapsible": true,
+    "collapsed": true
+}
```

Comment:
I thought it might be a better option to move "Storage Network" to a separate tab like Settings. This will make existing settings neater, thanks.

Reply:
Ok got it.

Reply:
fixed